### ISSUE TITLE
TEETH_SURFACE missing full enum list and left side d/m mapped backwards

### ISSUE THREAD

**Issue by dpwh24** [ctsit/imagemap#62]
# Issue 1

The enum list in the documentation for the TEETH_SURFACE option gives 1-16, a-j, and t-k. However, the imagemap also includes 17-32 which need to be added manually. Can these be added to the documentation for ease of use?

### Missing enum list
```
17r, 17r
17o, 17o
17l, 17l
17m, 17m
17d, 17d
17b, 17b
18r, 18r
18o, 18o
18l, 18l
18m, 18m
18d, 18d
18b, 18b
19r, 19r
19o, 19o
19l, 19l
19m, 19m
19d, 19d
19b, 19b
20r, 20r
20o, 20o
20l, 20l
20m, 20m
20d, 20d
20b, 20b
21r, 21r
21o, 21o
21l, 21l
21m, 21m
21d, 21d
21b, 21b
22r, 22r
22o, 22o
22l, 22l
22m, 22m
22d, 22d
22b, 22b
23r, 23r
23o, 23o
23l, 23l
23m, 23m
23d, 23d
23b, 23b
24r, 24r
24o, 24o
24l, 24l
24d, 24d
24m, 24m
24b, 24b
25r, 25r
25o, 25o
25l, 25l
25d, 25d
25m, 25m
25b, 25b
26r, 26r
26o, 26o
26l, 26l
26d, 26d
26m, 26m
26b, 26b
27r, 27r
27o, 27o
27l, 27l
27d, 27d
27m, 27m
27b, 27b
28r, 28r
28o, 28o
28l, 28l
28d, 28d
28m, 28m
28b, 28b
29r, 29r
29o, 29o
29l, 29l
29d, 29d
29m, 29m
29b, 29b
30r, 30r
30o, 30o
30l, 30l
30d, 30d
30m, 30m
30b, 30b
31r, 31r
31o, 31o
31l, 31l
31d, 31d
31m, 31m
31b, 31b
32r, 32r
32o, 32o
32l, 32l
32d, 32d
32m, 32m
32b, 32b
```

# Issue 2

Additionally, the indicators for d/m in the left side mappings are swapped. If you select 9m in the survey, 9d will be reported in the data. The order of the enum list doesn't appear to affect the mapping.

### In the left side upper quadrant, 9m and 10d are selected:
![teeth_map](https://user-images.githubusercontent.com/8475221/153926337-8f4d0bd1-0d8c-46c8-933a-ce7024d9ccd1.PNG)

### In the data, 9d and 10m are marked as checked:
![teeth_map_data](https://user-images.githubusercontent.com/8475221/153930857-6018c9f9-dbe8-4435-8984-80344e0503ab.PNG)

This same pattern persists for entire left side (9-16, f-j, o-k, 17-24)

Thanks

**Comment by ChemiKyle** [comment]
# Issue 2: 9m in the image maps to 9d in the checkboxes

Confirmed, all teeth images on the right half of the image ("Top Left - Quadrant 2" and "Bottom Left - Quadrant 3") are mirrored about the y axis, which wasn't accounted for in the map.

Note for testing: here is a minimal project xml (in a zip file because github doesn't like xml...) containing only the offending imagemap and a field that shows which items were selected

![image](https://user-images.githubusercontent.com/20332546/154103436-4990c26d-dee0-466f-b1c5-b42e2e0a9fe8.png)

[imagemap_teeth_surface.zip](https://github.com/ctsit/imagemap/files/8072779/imagemap_teeth_surface.zip)

**Comment by mbentz-uf** [comment]
Thanks for the complete list @dpwh24, all keys and enums were added to the documentation as well. 

Addressed by #63.
